### ISSUE TITLE
Hero carousel - Icons hidden

### ISSUE THREAD

**Issue by elavaud** [Wikiki/bulma-carousel#51]
Hi there,

bulma-carousel 3.0.0
vue-awesome 3.0.3

In the hero carousel, the navigation icons doesnt appear:

```
<section class="hero is-fullheight has-carousel">
  <div class="hero-carousel carousel-animated carousel-animate-fade">
    ...
    <div class="carousel-navigation is-overlay">

      <!-- Added as a matter of test, the one below will appear-->
      <icon name="chevron-left" aria-hidden="true"></icon>

      <div class="carousel-nav-left">
        <!-- However this one doesnt appear -->
        <icon name="chevron-left" aria-hidden="true"></icon>
      </div>

      ...
    </div>
  </div>
</section>
```

Any ideas?

**Comment by Wikiki** [comment]
Hi,

you're html is wrong: icons are based on fontawesome on the example (but can be anything else) so please check fontawesome is working well on your site.

**Comment by elavaud** [comment]
Thanks Wikiki,

Font awesome was well working on the site. However I was using [vue-awesome](https://github.com/Justineo/vue-awesome). All your carousel examples were well working except the hero-carousel one (I finally managed to get the icon displayed by adding "is-overlay" class, but then was not centered...).

But I actually switched to the official [vue-fontawesome](https://github.com/FortAwesome/vue-fontawesome) and now everything is working as expected.

Thanks!
